From 882b28e5a0d146fda9c4859b7232d14ae77bb518 Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Sun, 27 Mar 2022 08:40:06 -0400
Subject: [PATCH] erlang: don't install to target

This requires mix release configurations to copy ERTS to the release.
This isn't a big deal since it's been done for years in the Nerves
tooling, but just discarded when assembling the release since ERTS was
already on the target.
---
 package/erlang/erlang.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/erlang/erlang.mk b/package/erlang/erlang.mk
index 83a9e80374..dc160f8703 100644
--- a/package/erlang/erlang.mk
+++ b/package/erlang/erlang.mk
@@ -38,6 +38,7 @@ ERLANG_LICENSE_FILES = LICENSE.txt
 ERLANG_CPE_ID_VENDOR = erlang
 ERLANG_CPE_ID_PRODUCT = erlang\/otp
 ERLANG_INSTALL_STAGING = YES
+ERLANG_INSTALL_TARGET = NO
 
 # windows specific issue: https://nvd.nist.gov/vuln/detail/CVE-2021-29221
 ERLANG_IGNORE_CVES += CVE-2021-29221
-- 
2.25.1

